examples/listinvoices/README.md

```diff
@@ -0,0 +1,8 @@
+This example connects to LND and pulls 100 invoices every 15 seconds starting with the first.
+
+To use it, you will need to set the $LND_URL environment variable to the fqdn of your LND

```

Comment:
I think it'd probably be a good idea to avoid the abbreviation 'fqdn' to be more approachable, what do you think?

Reply:
Agreed. I’ll fix that.